engine/src/main/java/org/camunda/bpm/engine/impl/util/CallableElementUtil.java

```diff
@@ -40,9 +47,38 @@ public static ProcessDefinitionImpl getProcessDefinitionToCall(VariableScope exe
     String processDefinitionKey = callableElement.getDefinitionKey(execution);
     String tenantId = callableElement.getDefinitionTenantId(execution);
 
+    return getCalledProcessDefinition(execution, callableElement, processDefinitionKey, tenantId);
+  }
+
+  public static ProcessDefinition getStaticallyBoundProcessDefinition(BaseCallableElement callableElement, String tenantId) {
+    if(callableElement.hasDynamicReferences()){
+      return null;
+    }
+
+    VariableScope emptyVariableScope = new StartProcessVariableScope();
```

Comment:
Should we rather go for reusing the shared instance?

```suggestion
    VariableScope emptyVariableScope = StartProcessVariableScope.getSharedInstance();
```